### ISSUE TITLE
SplitButton loses focus during keyboard navigation

### ISSUE THREAD

**Issue by jbarton-egt** [primefaces/primeng#19365]
### Describe the bug

Keyboard navigation on splitbutton menu loses focus after selecting a MenuItem.

### Pull Request Link

https://github.com/primefaces/primeng/pull/19366

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/github-g1ck5gy7-bpnv7xme

### Environment

Angular 20
PrimeNG 20

### Angular version

20

### PrimeNG version

v20

### Steps to reproduce the behavior

1. Use keyboard (Tab key) to focus on the SplitButton menubutton
2. Use keyboard (Enter key, down arrow) to expand the menu, and highlight an option
3. Use keyboard (Enter key / space key) to select a menu item. Focus is lost, and pressing 'Tab' again will start the focus cycle back at the top of the screen.

### Expected behavior

After selecting a menu Item, I would expect the focus state to return to the menu trigger button in the SplitButton.

**Comment by cetincakiroglu** [comment]
fixed in v22
